install/Makefile

```
@@ -1,13 +1,34 @@
 STRIMZI_OPERATOR_NAMESPACE ?= openshift-operators
 KAFKA_CLUSTER_NAMESPACE ?= openshift-kafka-cluster
-PER_CLUSTER_MONITORING_NAMESPACE ?= kafka-monitoring-cluster
-GLOBAL_MONITORING_NAMESPACE ?= kafka-monitoring-global
+PER_CLUSTER_MONITORING_NAMESPACE ?= managed-services-monitoring
+GLOBAL_MONITORING_NAMESPACE ?= managed-services-monitoring-global
 KAFKA_PVC_SIZE ?= 10Gi
 RESOURCE_FILES_DIR ?= ./resources
 BUILD_DIR := build
 
 .PHONY: install/global/monitoring
 install/global/monitoring:
+  # create yaml files
+	@ mkdir -p ./$(BUILD_DIR)/monitoring-cluster
+	@ cp $(RESOURCE_FILES_DIR)/monitoring-cluster/operatorgroup.yaml ./$(BUILD_DIR)/monitoring-cluster/
+	@ cp $(RESOURCE_FILES_DIR)/monitoring-cluster/sub.yaml ./$(BUILD_DIR)/monitoring-cluster/
+	@ cp $(RESOURCE_FILES_DIR)/monitoring-cluster/grafana.yaml ./$(BUILD_DIR)/monitoring-cluster/
+	@ sed -i 's/<PER_CLUSTER_MONITORING_NAMESPACE>/$(PER_CLUSTER_MONITORING_NAMESPACE)/' ./$(BUILD_DIR)/monitoring-cluster/*.yaml
+	
+	# create namepsace
+	@ oc new-project $(PER_CLUSTER_MONITORING_NAMESPACE)
```

This fails on second run when the namespace already exists. You can use this to ignore the error:

`@ oc new-project $(PER_CLUSTER_MONITORING_NAMESPACE) | true`